src/models/Settings.js

```diff
@@ -173,7 +173,7 @@ export default Model.extend({
     piv: ['object', false, {}],
 
     //Email verify callback
-    stateTokenExternalId: 'string'
+    otp: 'string'
```

Comment:
We handle the `otp` as a query parameter (it may or may not have been clicked on from an email, customers do have some power to deliver the OTP code in alternate ways).  The app needs to read the `otp` parameter from the URL and then pass it to the widget to continue the flow. Made sense to me to have the config option match the query parameter name so there is no confusion on how to wire it up.